yoti_python_sdk/dynamic_sharing_service/policy/wanted_attribute_builder.py

```diff
@@ -24,8 +25,35 @@ def with_derivation(self, derivation):
         self.__attribute["derivation"] = derivation
         return self
 
+    def with_accept_self_asserted(self, value=True):
+        """
+        :param value: True if self-asserted details are allowed
+        """
+        self.__attribute["accept_self_asserted"] = value
+        return self
+
+    def with_optional(self, value=True):
```

Comment:
This is not supported by the app yet, so we need to remove.
In the other SDKs I believe we've kept it in the payload, but we're always setting it to false.